tests/test_basic.py

```diff
@@ -41,4 +41,4 @@ def test_spike(fh_http: ServerLayer):
     fh_http.stdout.seek(0)
     stdout = fh_http.stdout.read()
     assert "DENO: Got request body" in stdout
-    assert "RUST: modified request is" in stdout
+    #assert "RUST: modified request is" in stdout
```

Comment:
I removed it, because we agreed, that rust shouldn't print too much anymore, but currently I don't care.